ivadomed/testing.py

```diff
@@ -270,11 +275,13 @@ def run_inference(test_loader, model, model_params, testing_params, ofolder, cud
                     # Save merged labels with color
 
                     if pred_undo.shape[0] > 1 and ofolder:
-                        imed_visualize.save_color_labels(pred_undo,
-                                                     False,
-                                                     batch['input_metadata'][smp_idx][0]['input_filenames'],
-                                                     fname_pred.split(".nii.gz")[0] + '_color.nii.gz',
-                                                     slice_axis)
+                        logger.warning('No color labels saved due to a temporary issue. For more details see:'
+                                       'https://github.com/ivadomed/ivadomed/issues/720')
+                        # imed_visualize.save_color_labels(pred_undo,
+                        #                              False,
+                        #                              batch['input_metadata'][smp_idx][0]['input_filenames'],
+                        #                              fname_pred.split(".nii.gz")[0] + '_color.nii.gz',
+                        #                              slice_axis)
```

Comment:
same as comment above

Reply:
justified in justified in eca1c5f089e1a3f5a5fefe248a7746af10ef7fff